lib/pure/math.nim

```diff
@@ -48,6 +48,8 @@
 ## * `mersenne module<mersenne.html>`_ for Mersenne twister random number generator
 ## * `stats module<stats.html>`_ for statistical analysis
 ## * `strformat module<strformat.html>`_ for formatting floats for print
+## * `system module<system.html>`_ Some very basic and trivial math operators
+## are on system directly, to name a few ``shr``, ``shl``, ``xor``, ``clamp``, etc.
```

Comment:
From [the reStructuredText documentation](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#bullet-lists):

> List item bodies must be left-aligned and indented relative to the bullet; the text immediately after the bullet determines the indentation.

This change currently renders like this, which doesn't look good:

![output](https://user-images.githubusercontent.com/1811645/67130934-bd5feb80-f1b7-11e9-87be-ffa5f556d01c.png)

...so you should add two spaces here such that the 'a' in "are" lines up with the backtick:
```suggestion
##   are on system directly, to name a few ``shr``, ``shl``, ``xor``, ``clamp``, etc.
```

Yeah, documentation markup requires a lot of attention to small details like that, unfortunately...

Reply:
Thanks for taking the time to review, I was just observing the same, Github suggestion is cool.